packages/frontend-shared/src/gql-components/HeaderBarContent.cy.tsx

```diff
@@ -24,15 +26,23 @@ describe('<HeaderBarContent />', { viewportWidth: 1000, viewportHeight: 750 }, (
       .should('not.exist')
     })
 
+    cy.get('[data-cy="top-nav-browser-list-item"]')
+    .last().should('not.be.visible')
+    .scrollIntoView().should('be.visible')
+
     cy.contains('Version unsupported')
+    .scrollIntoView()
     .should('be.visible')
     .closest('[data-cy="top-nav-browser-list-item"]')
     .within(() => {
       cy.get('[data-cy="unsupported-browser-tooltip-trigger"]')
       .should('exist')
+      .trigger('mouseenter')
     })
 
-    cy.percySnapshot('after browsers open')
+    cy.contains('Unsupported browser')
+
+    cy.percySnapshot('unsupported browser tooltip')
```

Comment:
We probably need a visibility check to get the tooltip displayed before Percy takes the snapshot, it seems to take it too early - I think the content is is the DOM but the tooltip has not faded in yet. Some of our other tooltip Percy tests are a bit flaky and I'd guess it's for similar reasons.

Reply:
Added visibility check!